service/controllers/messages.go

```diff
@@ -221,6 +237,50 @@ func routes(routeObject response, w http.ResponseWriter) {
 				w.Write(jData)
 				TextToSpeech(response.Message + city + " " + state, 0)
 			}
+		} else if strings.ToLower(firstPars) == "meaning" {
+
+			if len(messageArr) == 0 {
+				w.Write([]byte(`{"status": "success", "message": "ENTER: meaning <word>", "result": ""}`))
+			} else {
+
+				word := messageArr[1:]
+				wordStr := strings.Join(word," ")
+
+				result := HandlerMeaning(wordStr)
+				response := processMeaning(result)
+				fmt.Println(len(response))
+
+				if len(word) == 1 && len(response) > 0 {
+
+					responseJSON := jsonResponseMeaning{
+					Status: true,
+					Message: "here is the meaning of the searched word",
+					Result: response,
+					}
+
+					jData, _ := json.Marshal(responseJSON)
+					w.Write(jData)
+					TextToSpeech(responseJSON.Message + " " + wordStr, 0)
+
+				} 
+				
+				if len(word) >= 2 || len(response) == 0 {
```

Comment:
this algorithm of implementation does not accept multiple words as a search in the scrapping? since googling multiple words would be a not-so-preferred situation

Reply:
I have changed the algorithm, and now it checks based on the response from the dictionary.py and had set it to len(response) > 0. Pls have a check.